### PR TITLE
Create members list

### PR DESCRIPTION
**Pull request by JamesVorder**
How does this look to you two?

### CODE REVIEW

**Review by stephen304 — APPROVED** [review]
lgtm. Would be good to also add push restrictions in the settings

**Comment by JamesVorder** [comment]
I added push restrictions, and added us all to the "Core" team. We alone have write access to this repo, but it is publicly visible. The push restrictions are a little crude... It is just restricted to getting two approvals before merge, not specifically linked to _who_ should do the approving. You're both admins, maybe you know a way to do that?

**Review by cure — APPROVED** [review]
LGTM thanks. I think you've done as much as possible with the push restrictions, they are not very sophisticated.

**Pull request merged**
